local/debian/Makefile

```diff
@@ -0,0 +1,27 @@
+IMG ?= saucelabs/forwarder:debian
+CONTAINER ?= forwarder-debian
+
+.PHONY: build
+build:
+	@podman build -t $(IMG) .
+
+.PHONY: package
+package: DIST ?= ../../dist/forwarder*linux_arm64.deb
+package:
+	@cd ../../ && ./bin/goreleaser release --snapshot --skip-docker --clean
```

Comment:
I was thinking maybe add soft link to git?

Reply:
Could link this work as a volume?

Reply:
I found the copy technique convenient to use, as it allows us to use pattern matching `*` (SNAPSHOT releases have commit hash in the file name). I doubt you can mount a volume without the knowledge of full file name.

Reply:
I see.

Reply:
The full file name is the name of the link. The question is can it read links.

Reply:
We can link it the same way we copy - with the `*`. 
```
.PHONY: package
package: DIST ?= ../../dist/forwarder*linux_arm64.deb
package:
	@cd ../../ && ./bin/goreleaser release --snapshot --skip-docker --clean
	@ln -s $(DIST) forwarder.deb
```

Reply:
Never mind. Soft links seem to be messing something with the Podman build 
```
make build  
Error: invalid symlink "/var/tmp/libpod_builder4117844942/build/forwarder.deb" -> "../../dist/forwarder_1.1-rc1-SNAPSHOT-d9b0b83.linux_arm64.deb"
gmake: *** [Makefile:6: build] Error 125
```